core/assets/bundles/bundle_vi.properties

```diff
@@ -1158,37 +1158,37 @@ block.message.name = Thông điệp
 block.illuminator.name = Đèn
 block.overflow-gate.name = Cổng tràn
 block.underflow-gate.name = Cổng tràn ngược
-block.silicon-smelter.name = Máy nấu silicon
-block.phase-weaver.name = Máy tạo Phase
-block.pulverizer.name = Máy nghiền
+block.silicon-smelter.name = Máy luyện silicôn
+block.phase-weaver.name = Máy may vải hợp tử
+block.pulverizer.name = Máy nghiền phế liệu
 block.cryofluid-mixer.name = Máy sản xuất chất làm lạnh
 block.melter.name = Lò nung chảy
 block.incinerator.name = Máy phân hủy
 block.spore-press.name = Máy nén bào tử
 block.separator.name = Máy phân tách
-block.coal-centrifuge.name = Máy tạo than
+block.coal-centrifuge.name = Máy ly tâm tạo than
```

Comment:
Orginal is enough to understand.
You make it harder to understand what it mean.